BREAKING.md

```diff
@@ -10,6 +11,16 @@
 `IProvideFluidDataStoreRegistry` implementation moved from IContainerRuntimeBase to IContainerRuntime. Data stores and objects should not have access to global state in container.
 `IProvideFluidDataStoreRegistry` is removed from IFluidDataStoreChannel - it has not been implemented there for a while (it moved to context).
 
+### NamedFluidDataStoreRegistryEntries 
+NamedFluidDataStoreRegistryEntries usage across repo is reduced substantially. Specifically, `ContainerRuntime.load()` is changed. Instead many interfaces are changed to accept IFluidDataStoreRegistry
+The following two classes are added to assist in conversion:
+`FluidDataStoreRegistry` class can be used to wrap input in many formats, including NamedFluidDataStoreRegistryEntries format into IFluidDataStoreRegistry. It takes Iterable<> of:
+1. old [name, Promise<factory>] format.
+2. [name, factory]
```

Comment:
The initial issue was that the input here forced us into creating a factory only object instead of an interface based object that was extendable. Are these "factory" objects now IFluidObjects?